src/main/java/at/hannibal2/skyhanni/features/nether/AtomHitBox.kt

```diff
@@ -0,0 +1,69 @@
+package at.hannibal2.skyhanni.features.nether
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.config.features.crimsonisle.AtomHitBoxConfig.AtomsEntries
+import at.hannibal2.skyhanni.data.IslandType
+import at.hannibal2.skyhanni.events.LorenzRenderWorldEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.EntityUtils
+import at.hannibal2.skyhanni.utils.EntityUtils.hasSkullTexture
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils.isInIsland
+import at.hannibal2.skyhanni.utils.RenderUtils
+import at.hannibal2.skyhanni.utils.RenderUtils.drawDynamicText
+import at.hannibal2.skyhanni.utils.getLorenzVec
+import net.minecraft.entity.item.EntityArmorStand
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import java.awt.Color
+
+@SkyHanniModule
+object AtomHitBox {
+
+    private val config get() = SkyHanniMod.feature.crimsonIsle.atomHitBox
+
+    @SubscribeEvent
+    fun onRender(event: LorenzRenderWorldEvent) {
+        if (!isEnabled()) return
+
+        for (mob in Atom.entries) {
+            if (mob.entry.isSelected()) {
```

Comment:
Simple optimisation I see is that you could do
```suggestion
        for (mob in config.atomsEntries) {
```

Reply:
I'am using the mob.texture, mob.displayName and color, I can't just use config.atomsEntries

Reply:
First of all you don't need to enums. You can just delete the one in config and add the data to ```Atom```. 
There is no issue with using a kotlin enum for a ```@ConfigEditorDraggableList```.